docs/creating.md

```diff
@@ -95,27 +95,40 @@ suitable to [autoupdate your project safely][the-copier-answersyml-file]:
 
 ### `_copier_conf`
 
-`_copier_conf` includes a representation of the current Copier
-[Worker][copier.main.Worker] object, also slightly modified:
+`_copier_conf` includes a slightly modified representation of the current Copier Worker
+object:
 
 -   It only contains JSON-serializable data.
 -   You can serialize it with `{{ _copier_conf|to_json }}`.
 -   ⚠️ It contains secret answers inside its `.data` key.
 -   Modifying it doesn't alter the current rendering configuration.
 
```

Comment:
How about deleting this entire snippet, as referring to the `Worker` class and the relationship between the render context and the `Worker` class is an implementation detail?